assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -187,6 +174,85 @@ def _save(
         metadata = fetch_mlflow_acft_metadata(base_model_name=self._model_id,
                                               is_finetuned_model=False,
                                               base_model_task=self._task)
+        if self._model_flavor == "OSS":
```

Comment:
we can keep common code in _save and move both OSS model save handling in a separate func as you've done hftv2